utils/inspect-response.js

```diff
@@ -1,6 +1,32 @@
 const explorer = require('./explorer');
-
+const config = require('../get-config')();
+const chalk = require('chalk');  // colorize output
 const util = require('util');
+
+
+const checkExistAccount = (error, options) => {
+    if(!String(error).includes('does not exist while viewing')) return false;
+
+    console.log(chalk`\n{bold.red Account {bold.white ${options.accountId}} is not found in {bold.white ${config.helperAccount}} network.\n}`);
+    
+    const re = new RegExp('[^\.]*$', 'gi');
```

Comment:
I'm not that picky, but Travis CI is. Only one issue left!
Please, change to `[^\.]*$` to ` [^.]*$`. That escape character is redundant.

P.S. You can check CI status in the `Checks` tab of the PR. Usually it's ready in 10-15 minutes after the commit.